packages/fxa-payments-server/src/routes/Checkout/index.tsx

```diff
@@ -243,6 +243,7 @@ export const Checkout = ({
       ]);
       setProfile(profile);
       setCustomer(customer);
+      setCoupon(coupon);
```

Comment:
I don't understand this call.

Reply:
left over from development - its unnecessary. removed.